static/scripts/login.js

```diff
@@ -12,7 +12,7 @@ $(document).ready(() => {
 		storage.local.clear();
 		storage.local.setItem('homepageVersion', newVersion.toString());
 	}
-
+    /*
```

Comment:
Why? This is logging to the browser's console, so what's the problem?

Reply:
true

Reply:
I want to reduce to over all console.log in our code, to find this parts that can effect our application. The data generated in server, 1KB more that we post out and that must consume by each user with this school traffic. If we really need it?